Finjector.Web/ClientApp/src/pages/Entry.tsx

```diff
@@ -42,10 +43,20 @@ const Entry = () => {
   });
 
   // in progress chart data
-  const [chartData, setChartData] = React.useState<ChartData>({
-    chartType: ChartType.PPM,
-    glSegments: buildInitialGlSegments(),
-    ppmSegments: buildInitialPpmSegments(),
+  const [chartData, setChartData] = React.useState<ChartData>(() => {
```

Comment:
@srkirkland do you prefer `React.useState()` to `useState()` ? i do the latter, should we pick one to be consistent?

Reply:
@laholstege I've always liked `React.useState()` mostly because it's more obvious what it is.  But i don't think i've ever been too consistent and I tend to use `useEffect()` and not `React.useEffect()`.

Reply:
What does your training class do?  I see the react docs mostly just use the function so that's one point for function only

Reply:
he does `React.useState()`, here's his opinion: 
>Personally, I don't want to have to fuss with imports. It's a tiny distraction to me, and I'd rather write a few extra characters every time. But this is 100% personal preference, you can manage this however you wish.

Reply:
here's my pitch for `useState()` 
1) `useEffect()` and `useState()` looks cooler than `React.useState()`
2) you can tell from the imports if a component has a state/effect/memo, which can be handy on long or complicated files 
3) `React.useState()` reminds me of the class i took where we had to do `React.createElement('div')` all the time 

... but that's all i got, i don't feel very strongly